mgmtfn/mesosplugin/contiv-isolator/Makefile

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Why do we need this makefile since we build contiv-isolator as part of top level makefile..